### DISCUSSION TITLE
Cannot find cmp command

### DISCUSSION THREAD

**Discussion by neolib** [msys2/MSYS2-packages#3026] [General]
I cannot get cmp command installed. https://packages.msys2.org/search returns an empty result when I search for 'cmp'. Please advise.

**Comment by mati865** [comment]
It's provided by [diffutils](https://packages.msys2.org/package/diffutils?repo=msys&variant=x86_64) package.

**Reply by neolib** [reply]
> It's provided by [diffutils](https://packages.msys2.org/package/diffutils?repo=msys&variant=x86_64) package.

I am always intimidated by pacman's commandline synax. For this case, how do you search what package provides cmp? I tried
`pacman -F cmp`
`pacman -Ss cmp`
but none of them works.

**Reply by mati865** [reply]
Since MSYS2 is running on Windows you have to add `.exe` suffix:
```
$ pacman -F cmp.exe
mingw32/mingw-w64-i686-diffutils 3.6-2
    mingw32/bin/cmp.exe
mingw64/mingw-w64-x86_64-diffutils 3.6-2
    mingw64/bin/cmp.exe
ucrt64/mingw-w64-ucrt-x86_64-diffutils 3.6-2
    ucrt64/bin/cmp.exe
clang32/mingw-w64-clang-i686-diffutils 3.6-2
    clang32/bin/cmp.exe
clang64/mingw-w64-clang-x86_64-diffutils 3.6-2
    clang64/bin/cmp.exe
msys/diffutils 3.8-3 (base-devel) [installed]
    usr/bin/cmp.exe
```

**Reply by neolib** [reply]
> Since MSYS2 is running on Windows you have to add `.exe` suffix:
> 
> ```
> $ pacman -F cmp.exe
> mingw32/mingw-w64-i686-diffutils 3.6-2
>     mingw32/bin/cmp.exe
> mingw64/mingw-w64-x86_64-diffutils 3.6-2
>     mingw64/bin/cmp.exe
> ucrt64/mingw-w64-ucrt-x86_64-diffutils 3.6-2
>     ucrt64/bin/cmp.exe
> clang32/mingw-w64-clang-i686-diffutils 3.6-2
>     clang32/bin/cmp.exe
> clang64/mingw-w64-clang-x86_64-diffutils 3.6-2
>     clang64/bin/cmp.exe
> msys/diffutils 3.8-3 (base-devel) [installed]
>     usr/bin/cmp.exe
> ```
OMG, I never knew that... Thanks.
